### DISCUSSION TITLE
null == undefined 의 결과와 이유를 설명해주세요

### DISCUSSION THREAD

**Discussion by pipisebastian** [publdaze/js-study#113] [궁금증]
- null == undefined 의 결과와 이유를 설명해주세요

**Comment by publdaze** [comment]
> - null == undefined 의 결과와 이유를 설명해주세요

- 동등하다고 판단하여 true, [자바스크립트 명세](https://262.ecma-international.org/11.0/#sec-abstract-equality-comparison)에 따라 단순히 true로 평가되는 것
